main.md

```diff
@@ -285,20 +294,21 @@ required by the application using SD-JWTs.
 
 ### `sd_digests` Claim (Digests of Selectively Disclosable Claims)
 
-An SD-JWT MUST include hash digests of the salted claim values that are included by the issuer
+An SD-JWT MUST include hash digests over the random values and the
+claim values that are included by the issuer
 under the property `sd_digests`. 
 
-The issuer MUST choose a unique and cryptographically random salt value
-for each claim value. Each salt value
+The issuer MUST choose a unique and cryptographically random value
+for each claim value. Each random value
 SHOULD contain at least 128 bits of pseudorandom data, making it hard for an
```

Comment:
This could be dependent on the digest derivation function, so perhaps move that information to the hash mechanism choice section (128 bits is ok for sha256 and hmac-sha256, but you could choose a smaller one if you pick a stronger iterative derivation function)

Reply:
moved to the security considerations section. and changed 128 bits to recommendation from the requirement (cc @danielfett )

Reply:
elaborated further in the latest commit.